python/lsst/ts/salobj/configurable_csc.py

```diff
@@ -480,7 +480,7 @@ def _get_default_config_dir(self):
     def add_arguments(cls, parser):
         parser.add_argument(
             "--configdir",
-            help="Directory containing configuration files for the start command.",
+            help="directory containing configuration files for the start command.",
```

Comment:
out of curiosity, why "directory" instead of "Directory"?

Reply:
to match the help text for built-in arguments such as `--help` and `--version`. I only recently realized they start with lowercase.